experimental/devices/mcp9808/mcp9808.go

```diff
@@ -0,0 +1,345 @@
+// Copyright 2018 The Periph Authors. All rights reserved.
+// Use of this source code is governed under the Apache License, Version 2.0
+// that can be found in the LICENSE file.
+
+package mcp9808
+
+import (
+	"encoding/binary"
+	"errors"
+	"sync"
+
+	"periph.io/x/periph/conn"
+	"periph.io/x/periph/conn/i2c"
+	"periph.io/x/periph/conn/mmr"
+	"periph.io/x/periph/conn/physic"
+)
+
+// Opts holds the configuration options.
+//
+// Slave Address
+//
+// Depending which pins the A0, A1 and A2 pins are connected to will change the
```

Comment:
Put this as a comment to `Addr` (I prefer short named, Addr, Res, more go style)